roles/pangea/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I am not sure what the intention is here but this is not going to be compatible across cloud providers. the default ssh user and hostnames will vary. Also, mantl-pangea is a private repository. This should really live somewhere else.